kiliautoml/utils/pytorchvision/trainer.py

```diff
@@ -23,17 +23,17 @@ def train_model_pytorch(
     model: nn.Module,
     dataloaders,
     epochs,
-    verbose=0,
+    verbose,
     class_names,
 ) -> Tuple[nn.Module, Dict[str, Any]]:
     """
     Method that trains the given model and return the best one found in the given epochs
     """
+    _ = verbose
```

Comment:
same here

Reply:
verbose is not used in this function but is declared in the base class. Since now why use logging, we no longer verbose argument in a bunch of places. Otherwise, flake8 would not be happy

Reply:
same answer